src/enter.js

```diff
@@ -26,6 +26,6 @@ export default class Enter extends Feature {
 		this.listenTo( editingView, 'enter', ( evt, data ) => {
 			editor.execute( 'enter' );
 			data.preventDefault();
-		} );
+		}, 'low' );
```

Comment:
OK, I can live with it. `low` makes some sense, cause indeed other listeners will usually want to precede it. And if anyone will want to do something after, there's still `lowest`.